src/ResourceManager/AzureBatch/Commands.Batch/Models.Generated/PSCloudPool.cs

```diff
@@ -364,7 +364,8 @@ public PSNetworkConfiguration NetworkConfiguration
                 this.networkConfiguration = value;
             }
         }
-        
+
+        [Obsolete("Use ResizeErrors instead")]
```

Comment:
@matthchr please update the warning to say something along the lines of

"The ResizeError property will be removed in an upcoming breaking change release. Please use the ResizeErrors property instead."